Sources/NIO/EventLoop.swift

```diff
@@ -465,6 +472,18 @@ extension EventLoop {
         return promise.futureResult
     }
 
+    /// Submit `task` to be run on this `EventLoop`.
+    ///
+    /// The returned `EventLoopFuture` will be completed when `task` has finished running. It will be identical to
+    ///  the `EventLoopFuture` returned by `task`.
+    ///
+    /// - parameters:
+    ///     - task: The synchronous task to run. As everything that runs on the `EventLoop`, it must not block.
+    /// - returns: An `EventLoopFuture` identical to the `EventLooopFuture` returned from `task`.
+    public func flatSubmit<T>(_ task: @escaping () throws -> EventLoopFuture<T>) -> EventLoopFuture<T> {
```

Comment:
the task closure returns a future so it shouldn't be throwing.

Reply:
Nice. Removed throws.